core/src/test/java/nl/nn/adapterframework/senders/JavascriptSenderTest.java

```diff
@@ -22,24 +22,23 @@
 import nl.nn.adapterframework.stream.Message;
 import nl.nn.adapterframework.testutil.ParameterBuilder;
 
-@RunWith(Parameterized.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JavascriptSenderTest extends SenderTestBase<JavascriptSender> {
 
-	@Parameterized.Parameter()
-	public JavaScriptEngines engine;
+	private final JavaScriptEngines engine = JavaScriptEngines.J2V8;
 
 	@Override
 	public JavascriptSender createSender() {
 		return new JavascriptSender();
 	}
 
-	@Parameters(name = "{0}")
-	public static Collection<Object[]> data() {
-		return Arrays.asList(new Object[][] {{JavaScriptEngines.J2V8}});
+	private Collection<JavaScriptEngines> data() {
+		return List.of(JavaScriptEngines.J2V8);
 	}
 
 	//Test without a given jsFunctionName. Will call the javascript function main as default
-	@Test
+	@ParameterizedTest
+	@MethodSource("data")
```

Comment:
Hier kan je beter een EnumSource van maken (met JavaScriptEngines als arg)

Reply:
Nice, wist niet dat, dat ook kon.
Het is aangepast